test/helpers/fixture_set.rb

```diff
@@ -126,6 +126,15 @@ def assert_service_account_present(name)
       assert desired.present?, "Service account #{name} does not exist"
     end
 
+    def assert_daemon_set_present(name)
+      found = false
+      daemon_sets = v1beta1_kubeclient.get_daemon_sets(namespace: namespace, label_selector: "app=#{name}")
+      daemon_sets.each do |ds|
```

Comment:
I'd suggest the same thing as we went with above for service accounts (`.find` + message on the assertion)